vm-virtio/src/vhost_user/handler.rs

```diff
@@ -30,6 +31,7 @@ pub struct VhostUserEpollHandler {
     pub vu_epoll_cfg: VhostUserEpollConfig,
     pub cvq: Option<CtlVirtqueue>,
     pub mem: Option<Arc<RwLock<GuestMemoryMmap>>>,
+    pub slave_req_handler: Option<MasterReqHandler<dyn VhostUserMasterReqHandler>>,
```

Comment:
Could you please squash that commit into your initial vhost-user-net commit?

Reply:
Ditto!